components/web/apps/org.wso2.carbon.apimgt.publisher/pom.xml

```diff
@@ -79,5 +163,10 @@
             </plugin>
         </plugins>
     </build>
-
+    <properties>
+        <maven.test.skip>false</maven.test.skip>
+        <npm.executable>npm</npm.executable>
```

Comment:
Thanks @roonyh , I have added the mvn profile for windows to change the npm executor command in this [commit](https://github.com/wso2/carbon-apimgt/pull/4029/commits/1d353597100bac8ff9ce313bc7f59a560b3457e0)